lib/matplotlib/axes/_base.py

```diff
@@ -275,10 +271,10 @@ def _setdefaults(self, defaults, *kwargs):
     def _makeline(self, x, y, kw, kwargs):
         kw = kw.copy()  # Don't modify the original kw.
         kwargs = kwargs.copy()
-        default_dict = self._getdefaults(None, kw, kwargs)
-        self._setdefaults(default_dict, kw, kwargs)
+        kw.update(kwargs)
```

Comment:
I have pretty extensive notes about keeping kw and kwargs distinct. I can almost guarantee you that this breaks stuff. All my notes about this can be found in the long comments nearby this area of code.